### PR TITLE
Conference Highlight Page Template

### PR DESCRIPTION
**Pull request by padmajabhol**

### CODE REVIEW

File: assets/css/conf_temp.css [hunk 1]
```diff
@@ -0,0 +1,211 @@
+@import url(//fonts.googleapis.com/css?family=Lato:300:400);
```
Comment (smaranjitghose) [inline assets/css/conf_temp.css:1]: Could you please use Inter or Poppins?

File: assets/css/conf_temp.css [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (smaranjitghose) [inline assets/css/conf_temp.css:188]: Remove the extra spaces please

File: experience.html [hunk 3]
```diff
@@ -55,111 +61,254 @@
...
-                  <path id="Vector_48" d="M160.89 54.1505H150.244C149.568 54.1505 149.02 54.6987 149.02 55.3749V55.5837C149.02 56.2599 149.568 56.808 150.244 56.808H160.89C161.566 56.808 162.114 56.2599 162.114 55.5837V55.3749C162.114 54.6987 161.566 54.1505 160.89 54.1505Z" fill="#bb86fc"></path>
```
Comment (smaranjitghose) [inline experience.html:89]: Avoid the linting/indentation changes for now

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
## General Feedback:

- Make sure to link the issue in the PR description
- Add a clip/snapshot of the changes in the PR description.

## Issue Specific Feedback:

- Please have a look at the reference provided in the Issue Description. The changes proposed by this PR do not remotely satisfy the requirements of the image slider UI requested.


If you wish to continue with the mentioned UI, do let us know

**Comment by padmajabhol** [comment]
> ## General Feedback:
> * Make sure to link the issue in the PR description
> * Add a clip/snapshot of the changes in the PR description.
> 
> ## Issue Specific Feedback:
> * Please have a look at the reference provided in the Issue Description. The changes proposed by this PR do not remotely satisfy the requirements of the image slider UI requested.
> 
> If you wish to continue with the mentioned UI, do let us know

The reference carousel had a different classname for each card , I talked about it to one of the project admins and he said it was fine if I wanted to work on a different UI but it was important to make the data dynamic. I can make the exact UI you asked for but it won't be able to render the data dynamically.

**Comment by anushbhatia** [comment]
Do mention this stuff in the description when you generate a PR so that PA's will know everything about it.

**Comment by padmajabhol** [comment]
> Do mention this stuff in the description when you generate a PR so that PA's will know everything about it.

I ll be more careful next time.

**Pull request closed without merging**